_posts/2020-11-12-quarkiverse.adoc

```diff
@@ -7,7 +7,7 @@ synopsis: Entering a world of extensions
 author: gastaldi
 ---
 
-Quarkus is a Kubernetes Native Java stack tailored for the OpenJDK HotSpot and GraalVM, crafted from the best of breed Java libraries and standards, and it is also a ecosystem of ever growing extensions. In the beginning we were adding all extensions to the core Quarkus repository but eventually it became too much of overhead or just not always sensible to include an extension into the core.
+Quarkus is a Kubernetes Native Java stack tailored for the OpenJDK HotSpot and GraalVM, crafted from the best of breed Java libraries and standards, as well as an ever growing ecosystem of its extensions. At the beginning we were accepting all the contributed extensions in the core Quarkus repository. Eventually though it grew too large and lead to a maintenance overhead. In addition to that, in some cases it's simply not always sensible to include an extension into the core Quarkus repository.
```

Comment:
Sorry, I meant to create a suggestion but created an actual commit :)

Reply:
No worries, thank you!

Reply:
I think we should percolate your changes to https://github.com/quarkiverse/quarkiverse/blob/master/docs/Home.md#welcome-to-quarkiverse too